### PR TITLE
Enable validating JWT

### PR DESCRIPTION
**Pull request by ibuildthecloud**
@hibooboo2

### CODE REVIEW

**Comment by hibooboo2** [comment]
Turn the auth on

**Comment by ibuildthecloud** [comment]
You turn it on

File: auth/auth.go [hunk 1]
```diff
@@ -1,19 +1,26 @@
...
+	tokenString := req.URL.Query().Get("token")
+
+	if len(tokenString) == 0 {
+		return false
+	}
 
-	token, err := jwt.ParseFromRequest(req, func(token *jwt.Token) (interface{}, error) {
-		return config.Config.Key, nil
+	token, err := jwt.Parse(tokenString, func(token *jwt.Token) (interface{}, error) {
+		return config.Config.ParsedPublicKey, nil
```
Comment (hibooboo2) [inline auth/auth.go]: This still does nothing. Please read https://github.com/dgrijalva/jwt-go/blob/master/jwt.go#L135 You need to compare the keys. They are never compared

**Comment by hibooboo2** [comment]
:+1: lgtm just need to test it.

**Pull request merged**